enf-reducer/cut-unnecessary-and.metta

```diff
@@ -0,0 +1,28 @@
+;(: AndCut (-> Expression Expression))
+(= (AndCut $expr)
+    (let*
+        (
+            (($gSet $child) (getGsetAndChildren $expr))
+            ($tail (cdr-atom $child))
+        )
+        (if (and (== $tail ()) (== $gSet ()))
+            (let*
+                (
+                    (($Grandgset $GrandChildren) (getGsetAndChildren (car-atom $child)))
+                    ($allLiterals (collapse (getGuardSet (superpose $GrandChildren) ())))
+                    ($test (collapse (let $a (getGsetAndChildren (superpose $GrandChildren)) (car-atom $a))))
```

Comment:
What is this line doing here? I don't see the `$test` variable being used else where?